Formula/percona-toolkit.rb

```diff
@@ -5,9 +5,9 @@ class PerconaToolkit < Formula
 
   desc "Percona Toolkit for MySQL"
   homepage "https://www.percona.com/software/percona-toolkit/"
-  url "https://www.percona.com/downloads/percona-toolkit/3.2.0/source/tarball/percona-toolkit-3.2.0.tar.gz"
-  sha256 "e7c7028631d5b1c66ba21841c55919537bd4ed2f3a5975f989fa4abd7d9d753d"
-  license "GPL-2.0"
+  url "https://www.percona.com/downloads/percona-toolkit/3.2.1/source/tarball/percona-toolkit-3.2.1.tar.gz"
+  sha256 "92da0d3712533205f283515ff47ce68521fc2de20989c45b4c65a9743d0f7487"
+  license any_of: ["GPL-2.0-only", "Artistic-1.0-Perl"]
```

Comment:
Why `Artistic-1.0-Perl`?

Did it specify a version for the artistic license? If not, should we go with `Artistic-1.0+` to allow any artistic license?

Reply:
```
On Debian systems, the complete text of the Artistic License can  be found in
`/usr/share/common-licenses/Artistic'.
```

On debian systems, the common license is the perl edition.

Reply:
Ah! Didn't realize that!